src/Microsoft.Azure.ServiceBus/Core/IMessageReceiver.cs

```diff
@@ -97,6 +97,7 @@ public interface IMessageReceiver : IReceiverClient
         /// only when <see cref="ReceiveMode"/> is set to <see cref="ServiceBus.ReceiveMode.PeekLock"/>.
         /// </remarks>
         /// <param name="lockTokens">An <see cref="IEnumerable{T}"/> containing the lock tokens of the corresponding messages to complete.</param>
+        /// When using AMQP, this operation can only be performed on messages that were received by this receiver.
```

Comment:
We may not need to mention `When using AMQP`. That's the only one supported right now